.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,112 @@
+name: Continuous integration
+on: [push]
+env:
+  TOX_PARALLEL_NO_SPINNER: 1
+  PYTEST_ADDOPTS: -x
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.5-alpine
+        ports:
+        - 5432:5432
+      elasticsearch:
+        image: hypothesis/elasticsearch:latest
+        ports:
+        - 9200:9200
+        env:
+          discovery.type: single-node
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
+    - name: Checkout git repo
+      uses: actions/checkout@v2
+    - name: Create test databases
+      run: psql -U postgres -h localhost -p 5432 -c 'CREATE DATABASE htest'
+    - name: Cache the .tox dir
+      uses: actions/cache@v2
+      with:
+        path: |
+          .tox/checkformatting
+          .tox/tests
+          .tox/coverage
+        key: ${{ runner.os }}-tox-backend-${{ hashFiles('tox.ini') }}-${{ hashFiles('requirements*') }}-${{ hashFiles('setup.py') }}-${{ hashFiles('setup.cfg') }}
+        restore-keys: |
+          ${{ runner.os }}-tox-backend-
+    - name: Run tox
+      run: tox --parallel auto -e checkformatting,tests,coverage
```

Comment:
A change here compared to Jenkins is that the tests are running in an Ubuntu VM rather than the Alpine Linux docker environment. This creates a small possibility of environment differences causing tests to pass in CI but the Docker image not to work as expected. Is there a plan for this? Rely on QA? Set up an additional build in master that runs on Alpine?

Reply:
Yep, that is a change. I think we could run the tests in Docker/Alpine on GHA but I think it'd make the yml file more complicated and would be slower, so I don't think it's worth it. I don't think we've ever had a "it works on Ubuntu but not on Alpine" problem.

One thing we should probably do (if we aren't already) is run our apps in a Python virtualenv in Alpine to isolate them from Alpine's system Python packages.

But the plan was just to ignore this to be honest!

Reply:
> But the plan was just to ignore this to be honest!

I think that's a reasonable decision given the perceived level of risk, but I just wanted to make sure it was an explicit choice.